tests/integration/integration_test.go

```diff
@@ -699,9 +699,10 @@ status: {}
 
 			By("creating the artifacts")
 			Expect(files).To(ContainElements(
-				"nginx/profile-installation.yaml",
-				"nginx/artifacts/bitnami-nginx/HelmRelease.yaml",
-				"nginx/artifacts/bitnami-nginx/HelmRepository.yaml",
+				"artifacts/bitnami-nginx/ConfigMap.yaml",
+				"profile-installation.yaml",
+				"artifacts/bitnami-nginx/HelmRelease.yaml",
+				"artifacts/bitnami-nginx/HelmRepository.yaml",
```

Comment:
is this missing a check to see if the configmaps correct?

Reply:
🤔  i can probably grab the deployed pod and check that the applied spec is correct? does feel a bit like we are testing a well-known feature of theirs tho

Reply:
i will think about it, and merge what we have now so you are unblocked

Reply:
> thinking i can probably grab the deployed pod and check that the applied spec is correct? does feel a bit like we are testing a well-known feature of theirs tho

True... but not testing it also sort of feels wrong :shrug: